noir-projects/noir-contracts/contracts/test_contract/src/test/utils.nr

```diff
@@ -0,0 +1,24 @@
+use dep::aztec::{
+    prelude::AztecAddress, test::helpers::test_environment::TestEnvironment,
+};
+
+pub fn setup() -> (
+  &mut TestEnvironment,
+  AztecAddress,
+  AztecAddress
+) {
+    let mut env = TestEnvironment::new();
+    let account_1 = env.create_account();
+
+    env.impersonate(account_1);
+
+    let test_contract = env.deploy("./@test_contract", "Test").without_initializer();
+    let test_contract_address = test_contract.to_address();
+    env.advance_block_by(1);
+
+    (
+      &mut env,
+      test_contract_address,
+      account_1
+    )
+}
```

Comment:
Why create a whole new module and two separate files for a setup function if we'll only call it once? The same can be asked of most other tests, this is quite boilerplatey.